src/EnergyPlus/EconomicTariff.cc

```diff
@@ -482,11 +485,67 @@ namespace EconomicTariff {
 				tariff( iInObj ).energyConv = 9.4781712e-9;
 				tariff( iInObj ).demandConv = 0.00003412;
 			} else {
-				tariff( iInObj ).convChoice = conversionKWH;
-				tariff( iInObj ).energyConv = 0.0000002778;
-				tariff( iInObj ).demandConv = 0.001;
-				ShowWarningError( RoutineName + CurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\" invalid data" );
-				ShowContinueError( cAlphaFieldNames( 3 ) + "=\"" + cAlphaArgs( 3 ) + "\", Defaulting to KWH." );
+
+
+				// TODO: is this needed?
```

Comment:
I like clarity, just wasn't used to this method. If it had said "For reviewer: is this needed?" I would have got it immediately.  So if tariff.kindWaterMtr is already 0, then this is not needed. BUT, initializing to an explicit is not wrong either. So I think this is easier regarding maintenance than searching for the constructor default.